src/utils/firebase_list_observable.ts

```diff
@@ -40,9 +40,14 @@ export class FirebaseListObservable<T> extends Observable<T> {
     });
   }
 
-  remove(item:FirebaseOperation): Promise<void> {
+  remove(item:FirebaseOperation = null): Promise<void> {
     // TODO: remove override when typings are updated to include
     // remove() returning a promise.
+    
+    // if no item parameter is provided, remove the whole list
+    if (!item) {
```

Comment:
What if a non-typescript user passes `0` as a key? Do we want this method to allow vanilla JavaScript type coercion from a Number to a String so people can remove items with key `"0"`?